bash/update_package_properties.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -euxo pipefail
```

Comment:
I'd prefer if we had `set +x` and see the result of each line in the output. This is a matter of preference, and `set -x` creates definitely cleaner outputs. However if things go wrong, this is harder to debug

Reply:
I think you're confusing `-x` and `+x`. `set -x` echos the command to `stderr`, which is what this is using and it sounds like the thing you want.

Reply:
Yes, I was confused here. Thanks for pointing it out

Reply:
Can I mark as resolved if you don't have any further recommendations?